smoke-test.bash

```diff
@@ -19,3 +19,4 @@ while true; do
 done
 
 echo "Smoke test succeeded"
+docker compose down
```

Comment:
Should do this with `trap` so it runs when the test fails somewhere.

Reply:
I have thought of it. But I think we should keep the containers running if the test fails so that we can debug conveniently